bin/tessel-2.js

```diff
@@ -115,7 +115,8 @@ parser.command('crash-reporter')
   .option('test', {
     flag: true,
     help: 'Test the Crash Reporter.'
-  });
+  })
+  .help('Crash Reporter enable/disable');
```

Comment:
A few lines above it there is `--on` and `--off` flags. Not sure how to get the current state, or if there is a way or not.

Reference: https://github.com/tessel/t2-cli/blob/master/bin/tessel-2.js#L107-L114